hasura/graphql-engine
# Support auto suggest for default values of supported data types

Fix #1591 
### Description


Autocomplete user input on default value based on the pg data.

### Affected components 


- Console

### Related Issues




### Solution and Design


1) Retrieves list of functions supported by postgres by applying the following filter
  - Fetch functions whose return type is a primitive data type like text, varchar etc,
  - Filter the types which belongs to `Pseudo` category
2) Infers default values of types based on their type cast target types

### Steps to test and verify


1) Navigate to create table
2) Add a column called `dob` with column type `date`. The default input should provide list of suggestions
..
1) Navigate to modify and add/modify column

### Limitations, known bugs & workarounds
